### ISSUE TITLE
Third plot is confusing

### ISSUE THREAD

**Issue by sarah-gao** [mfhamir/apple_mobility_2021#13]
Hey @mfhamir I would revisit your third plot as it's unclear what the plot is trying to show. I think even just changing the title would make it more clear. It would also be helpful to be more specific in your comments for this plot what you're trying to show / do.
https://github.com/mfhamir/apple_mobility_2021/blob/8c6aff5cc32b1a17caf6faf5c84436b87dbb4653/Analysis.Rmd#L138

**Comment by mfhamir** [comment]
0b3b14a8d36269e0e5551d5ccc9b720346b0948e - just made sure that the plot was fixed, and also did make knitting fixes when i did the WSL script before and I had to make a change to my analysis.RMD so I think that's ok, I added new commits of my README and analysis file

**Comment by sarah-gao** [comment]
Hey @mfhamir I think this third plot is still unclear. In plain English, can you explain what you're trying to show? It looks like it's an aggregate of relative mobility across time by city and county categorized by transportation type, but if you think about it, this doesn't seem like super useful information. I would suggest paring it down and simplifying so that you focus on maybe one or two variables at a time.

**Comment by mfhamir** [comment]
Hi Sarah,

I will revisit the plot, I think initially the data set it asks us to
analyze is not that detailed so it was hard to really analyze something
that different to what the other graphs did but I will re-do it and submit
a new version later this week tonight. I will also look to fix the other
errors.

Thank you for your feedback.

**Comment by mfhamir** [comment]
later tonight*

**Comment by sarah-gao** [comment]
@mfhamir I think the changes in the labels is helpful and makes the plot make more sense. However, I would encourage you to think carefully with your final project what exactly you're trying to analyze / show in your figures as I still think that what you're showing with this third figure is not super informative. I'll close this issue for now but encourage you to keep that in mind for your final project.
